Openstack_antelope_installation/maas_and_juju_install_and_configure/configure_maas.yaml

```diff
@@ -0,0 +1,102 @@
+- name: Configure MAAS
+  hosts: localhost
+  vars_files:
+    - config.yaml
+
+  tasks:
+    - name: Login to MAAS
+      shell: |
+        export APIKEY=$(sudo maas apikey --username {{MAAS_USR}})
+        maas login {{MAAS_USR}} 'http://{{MACHINE_IP}}:5240/MAAS/' $APIKEY
+      ignore_errors: yes
+
+    - name: set DNS
+      shell: |
+        maas {{MAAS_USR}} maas set-config name=upstream_dns value={{DNS_SERVER}}
+
+    - name: enable proxy
+      shell: |
+        maas "{{MAAS_USR}}" maas set-config name=enable_http_proxy value=true
+
+    - name: set proxy
+      shell: |
+        maas "{{MAAS_USR}}" maas set-config name=http_proxy value="{{http_proxy}}"
+
+    - name: set NTP
+      shell: |
+        maas "{{MAAS_USR}}" maas set-config name=ntp_servers value={{NTP_SERVER}}
+
+    - name: Generate SSH key pair
```

Comment:
We should use existing ansible modules over the command or shell modules, wherever possible:
https://docs.ansible.com/ansible/latest/collections/community/crypto/openssh_keypair_module.html